index.php

```diff
@@ -466,6 +466,7 @@
 
 	$content = mb_convert_encoding($a->page["content"], 'HTML-ENTITIES', "UTF-8");
 
+	/// @TODO one day, kill those error-surpressing @ stuff, or PHP should ban it
```

Comment:
I wholly agree with you in general,, but here I don't see any obvious test to make sure `loadHTML` doesn't output any notice/error message.